crates/holochain/src/conductor/tests/network_info.rs

```diff
@@ -32,4 +35,46 @@ async fn network_metrics() {
 
     let network_metrics = &network_metrics[dna.dna_hash()];
     assert_eq!(network_metrics.gossip_state_summary.peer_meta.len(), 2);
+
+    let fake_dna = fake_dna_hash(1);
+    let response = conductors[0]
+    .dump_network_metrics_for_app(
+        &app_id,
+        Kitsune2NetworkMetricsRequest {
+            dna_hash: Some(fake_dna),
+            include_dht_summary: false,
+        },
+    )
+    .await;
+    assert!(response.is_err());
+
+    // Create a disabled clone cell for app1
+    let clone_cell = conductors[0]
+        .create_clone_cell(
+            &app_id,
+            CreateCloneCellPayload {
+                role_name: dna.dna_hash().to_string(),
+                modifiers: DnaModifiersOpt::none().with_network_seed("test_seed".to_string()),
+                membrane_proof: None,
+                name: Some("disabled_clone".into()),
+            },
+        )
+        .await
+        .unwrap();
+    
+    let clone_cell_id = CloneCellId::CloneId(clone_cell.clone_id);
+    let response = conductors[0].clone().enable_clone_cell(&app_id, &EnableCloneCellPayload{ clone_cell_id}).await;
+    assert!(!response.is_err());
+
+    let response = conductors[0]
+    .dump_network_metrics_for_app(
+        &app_id,
+        Kitsune2NetworkMetricsRequest {
+            dna_hash: Some(clone_cell.cell_id.dna_hash().clone()),
+            include_dht_summary: false,
+        },
+    )
+    .await;
+    assert!(!response.is_err());
```

Comment:
```suggestion
    assert!(response.contains_key(clone_cell.cell_id.dna_hash()));
```
Something like this, similar to line 34 above maybe, to check that it also returns info for the correct dna?